### DISCUSSION TITLE
Special Areas of Countries

### DISCUSSION THREAD

**Discussion by RiemannianSheldor** [electricitymaps/electricitymaps-contrib#4908] [Ideas]
Hi,
I Wonder if it is possible to see the data of the federal states of germany in a similar way as in Italy. It would be nice to switch between whole-country-stats and region-stats.

sincerely

**Comment by VIKTORVAV99** [comment]
Currently we use the ENTSO-E Transparency Platform to get data from Germany and Italy but the difference is that Italy is divided into different bidding areas. This means that the ENTSOE-E Transparency Platform has data for each individual bidding zone which in turn means we can get the data for all the individual zones and the exchange volumes between them.

As it stands right now we have not found a source for Germany that provide all the data we need to split it up so it will have to stay as a country for now. There is however a [proposal](https://www.acer.europa.eu/sites/default/files/documents/Individual%20Decisions/ACER%20Decision%2011-2022%20on%20alternative%20BZ%20configurations.pdf) from ACER that aims to split up Germany in multiple bidding zones. If that is done there should be no issues adding it to our map.

[Annex I (with maps)](https://www.acer.europa.eu/sites/default/files/documents/Individual%20Decisions_annex/ACER%20Decision%2011-2022%20on%20alternative%20BZ%20configurations%20-%20Annex%20I%20-%20rectified.pdf)
